fr/UserGuide/_sidebar.md

```diff
@@ -12,15 +12,18 @@
     - [Importer le stockage AWS](/fr/UserGuide/Storage/Importer-le-stockage-AWS.md)
     - [Importer le stockage de Google Cloud Platform](/fr/UserGuide/Storage/Importer-le-stockage-GCP.md)
     - [Accédez à votre compte de stockage dans Databricks](/fr/UserGuide/Databricks/Accédez-à-votre-compte-de-stockage-dans-Databricks.md)
-    - [Ajouter une interface utilisateur dans Databricks](https://learn.microsoft.com/en-us/azure/databricks/ingestion/add-data/)
+    - [Ajouter une interface utilisateur dans Databricks](/fr/https://learn.microsoft.com/en-us/azure/databricks/ingestion/add-data/)
 
   - Databricks et Notebooks Python
     - [Qu'est-ce qu'Azure Databricks ?](/fr/UserGuide/Databricks/Databricks.md)
     - [Comment intégrer Git dans Databricks](/fr/UserGuide/Databricks/Intégration-de-Git.md)
     - [Comment utiliser l'extension VS Code avec Databricks](/fr/UserGuide/Databricks/extension-vscode.md)
-    - [Intégrer Git dans Databricks](/fr/UserGuide/Databricks/Intégration-de-Git.md)
     - [Comprendre les politiques de cluster](/fr/UserGuide/Databricks/Politiques-des-clusters.md)
 
+  - Utiliser PostgreSQL sur DHSF
+    - [Créer et utiliser une base de données PostgreSQL sur DHSF](/fr/UserGuide/Database/Postgres.md)
+    - [Comparer PostgreSQL et Databricks](/fr/UserGuide/Database/Comparaison-des-banques-de-données-psql.md)
+
```

Comment:
All is good except the file name... 'banques de donnees' should be 'base de donnees'